.azure-pipelines/client.yml

```diff
@@ -2,6 +2,8 @@
 #  'build_targeting_string' comma separated glob strings selecting the packages that should be built. EG: "azure-keyvault,azure-mgmt-batch" or "azure-*"
 #  'codecov-python-repository-token' Is the codecov.io repository token.
 #  'long_running_tests' Is a boolean indicating that the long-running tests should be executed. 
+#  'EnableDetectorPip': true . Required for Component Governance Support
```

Comment:
This is required for the Component Governance Support

Reply:
Do you know more about this variable?  Why does CG require it to be set?  What happens if it's not set?  Why doesn't CG just default this internally to "true", instead of requiring everyone to set it?

Reply:
If we've verified this is required for CG support, I'm fine merging this PR as-is, but we should followup with the CG team.